### DISCUSSION TITLE
OSD Modifications - Way to remove the speaker icon from Volume Bar

### DISCUSSION THREAD

**Discussion by advice2020** [mpv-player/mpv#14953] [General]
I was wondering if it was possible to hide/remove the speaker icon that shows up on the left side of the Volume Bar OSD?
I know that to customize the default OSC, you have to get the osc.lua file from the source code but I have never seen any reference to anything similar for customizing OSDs.  Does anyone know if a similar file exists for OSDs?  I imagine this is where that speaker icon would be?

Thank You

**Comment by hooke007** [comment]
> Does anyone know if a similar file exists for OSDs

No

**Comment by advice2020** [comment]
Okay thank you, do you know if it is possible to remove or replace the speaker icon to the left of the volume bar?
